test/langtools/tools/javac/diags/examples/UnderscoreInLambdaExpression.java

```diff
@@ -21,9 +21,7 @@
  * questions.
  */
 
-// key: compiler.misc.feature.unnamed.variables
-// key: compiler.warn.preview.feature.use.plural
-// options: --enable-preview -source ${jdk.version} -Xlint:preview
+// options: -Xlint:preview
```

Comment:
`-Xlint:preview` is probably unnecessary here.

Reply:
If I remove everything, the example does not work. I will leave a `-source 21`